infoschema/slow_log.go

```diff
@@ -319,6 +329,12 @@ func (st *slowQueryTuple) setFieldValue(tz *time.Location, field, value string)
 			return errors.AddStack(err)
 		}
 		st.memMax = num
+	case variable.SlowLogSucc:
+		succ, err := strconv.ParseBool(value)
+		if err != nil {
+			return errors.AddStack(err)
```

Comment:
you can use `errors.Wrap()` to explain which operation causes this error.

Reply:
Great, thanks, Done. 
I add `errors.Wrap()` outside of this function.

Reply:
I think it's better to wrap it here? Why delaying it to the outside?

Reply:
done.